packages/@uppy/status-bar/src/StatusBar.js

```diff
@@ -234,12 +234,28 @@ const ProgressBarProcessing = (props) => {
 
 const ProgressDetails = (props) => {
   return <div class="uppy-StatusBar-statusSecondary">
-    { props.numUploads > 1 && props.i18n('filesUploadedOfTotal', { complete: props.complete, smart_count: props.numUploads }) + ' \u00B7 ' }
-    { props.i18n('dataUploadedOfTotal', {
-      complete: prettyBytes(props.totalUploadedSize),
-      total: prettyBytes(props.totalSize)
-    }) + ' \u00B7 ' }
-    { props.i18n('xTimeLeft', { time: prettyETA(props.totalETA) }) }
+    {
+      props.numUploads > 1 &&
+      props.i18n('filesUploadedOfTotal', {
+        complete: props.complete,
+        smart_count: props.numUploads
+      }) + ' \u00B7 '
+    }
+    <span class="uppy-StatusBar--onlyForDesktop">
```

Comment:
Nitpick: I would maybe call this `uppy-StatusBar--additionalInfo` or something, so that the desktop/mobile abstraction is hidden. Then devs using Uppy can hide this additional info based on a preference or custom condition. And we can later decide to show this on mobile, on a second/third line for instance, while still keeping the class name.